tutorials/terminology.md

```diff
@@ -63,10 +63,10 @@ to developers touching the source code.
     the world, while the **primary** runner is keeping all secondaries in sync.
     Worlds that are not split across runners don't have a primary runner.
 
-* **[Event manager](classignition_1_1gazebo_1_1EventManager.html)**:
+* \ref gz::sim::EventManager "Event manager":
     Manages events that can be sent across systems and the server. Plugins can
     create and emit custom
-    [Event](https://gazebosim.org/api/common/3.0/classignition_1_1common_1_1Event.html)s
+    [Event](https://gazebosim.org/api/common/5/classgz_1_1common_1_1Event.html)s
```

Comment:
Can't you use `\ref gz::common::Event` here?

Reply:
How can I test these `\ref` links? Github previewed markdown does not set them correctly.

Reply:
I agree you should be able to use `ref gz::common::Event`

@quarkytale, you can build the docs locally by running `make doc` in the build directory. You'll need to  have doxygen installed. This will create a `doxygen` directory that contains an `index.html`.

Reply:
That worked.

(We talked about how to test offline, and I added the instructions to test in the main description since)